.coveragerc

```diff
@@ -0,0 +1,4 @@
+[run]
+omit =
+    # omit everything under plugins for now
+    pymarlin/plugins/*
```

Comment:
is newline required?

Reply:
doesn't seem like it, getting a valid report omitting plugins with it.

if you're talking about the preceeding new line, i'm doing it this way as following other example but the config rc may use the indentation show it as a block - and we may want to omit more (or include more) in the future like this.